lib/scope.js

```diff
@@ -1742,7 +1742,13 @@ module.exports = {
     input: async function ( scope, { handle, answer }) {
       /* Set value of some `input` element to the given value. */
 
-      let type = await handle.evaluate( elem => { return elem.getAttribute('type'); });
+      let type = await handle.evaluate( elem => { 
+        if (elem) {
+          return elem.getAttribute('type'); 
+        } else {
+          return '';
+        }
```

Comment:
I don't know what it should do. I had to add this fix because `elem` was `null`, just to avoid running into uncertain null errors. Since we don't know the context of the error fully, it's likely something with puppeteer (or our use of puppeteer), and I'd rather just not have that happen. Throwing another error IMO would be the same issue there, we'd have to catch the error everywhere to else to prevent errors propagating.

I'd be willing to add a log, though it can't go right there (because that's JS that's executing in the browser, not this machine).

Reply:
Did a little experiment in the browser. When a type is null, the default type is 'text', so we'll use that instead of an empty string. I can do that when I'm working on peeling out the radio button stuff.